base/uk.ac.stfc.isis.ibex.opis/resources/fermi_chopper.opi

```diff
@@ -2265,27 +2265,27 @@ $(pv_value)</tooltip>
           </font>
           <forecolor_alarm_sensitive>false</forecolor_alarm_sensitive>
           <foreground_color>
-            <color name="ISIS_OPI_Foreground" red="192" green="192" blue="192"/>
+            <color name="ISIS_OPI_Foreground" red="192" green="192" blue="192" />
           </foreground_color>
           <height>25</height>
           <name>LED</name>
           <off_color>
-            <color name="ISIS_Red_LED_Off" red="102" green="0" blue="0"/>
+            <color name="ISIS_Red_LED_Off" red="102" green="0" blue="0" />
           </off_color>
           <off_label>OFF</off_label>
           <on_color>
-            <color name="ISIS_Red_LED_On" red="255" green="100" blue="100"/>
+            <color name="ISIS_Emergency_Button_Background" red="255" green="0" blue="0" />
```

Comment:
(First instance) Still appropriate to use the **ISIS_Red_LED_On** name for the <color> tag to reflect what the LED does.

Reply:
Made the change